static/scripts/topicEdit.js

```diff
@@ -111,13 +112,19 @@ class TopicBlockWrapper extends React.Component {
                             />
 
                             <div className="input-group-btn">
-                                <button className="btn btn-secondary dropdown-toggle" type="button" data-toggle="dropdown">
+								<button
+									aria-label={$t('global.label.settings')}
+									className="btn btn-secondary dropdown-toggle"
+									type="button"
+									data-toggle="dropdown">
                                     <i className="fa fa-cog"></i>
                                 </button>
                                 <div className="dropdown-menu dropdown-menu-right">
-                                    <a className="dropdown-item text-danger" onClick={this.onRemoveWithCallback.bind(this)}>
+                                    <button
+										className="dropdown-item text-danger"
+										onClick={this.onRemoveWithCallback.bind(this)}>
                                         <span><i className="fa fa-trash" /> {$t('global.button.remove')}</span>
```

Comment:
I think we can add aria-hidden to the icon because we have a visual label.

Reply:
done